components/gallery/GalleryItemTabsPanel/GalleryItemActivity.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="gallery-activity-events-wrapper is-flex is-flex-direction-column">
-    <div class="events p-5 is-flex is-flex-direction-column">
+    <div class="events py-5 px-6 is-flex is-flex-direction-column">
```

Comment:
Isn't it better to use p-5?

Reply:
`px` `py` value is different. one is `5` and one is `6`